### PR TITLE
Add instructions for preparing a power cut

### PR DESCRIPTION
**Pull request by lpetre-ulb**
Add the instructions for preparing a power cut as it was done at ULB on 19 November 2018.

Also add precisions about the recovery from a power cut. Some services must run on the DAQ machine and AMC's must be out of extraction mode.

### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context
We might need to safely power off a GEM stand in case of a planned power cut.

## Checklist:
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: v3ElectronicsUserGuide.md [hunk 1]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
...
 To recover a GEM test stand after a power cut execute:
 
- 1. Ensure the uTCA crate and associated hardware all have power
-     - E.g. the crate, all AMCs, network swtiches, DAQ computer, etc...
- 2. Enter the AMC13 tool and enable clocks to the AMC of interest by following instructions under [Enabling Clock to an AMC Slot](#enabling-clock-to-an-amc-slot),
- 3. For each CTP7 login as `texas` and execute: `killall rpcsvc`
+ 1. Ensure the uTCA crate and associated hardware all have power.
+     - E.g. the crate, network switches, DAQ computer, etc...
+ 2. Start the DAQ computer first, then:
+     - Ensure that the `sysmgr`, `xinetd` and `dnsmasq` services are correctly started. You can use the `systemctl status <service name> command to check each service status. The `Active` field must report `active (running)`.
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: you're missing the closing ` here, e.g. you have:

```
`systemctl status <service name>
```

Should be 

```
`systemctl status <service name>`
```

File: v3ElectronicsUserGuide.md [hunk 2]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
     - Note this will kill any running process on the hardware, but if you're updating FW no one should be using the system anyway.
  31.  Summarize the actions you took in the elog entry you have already opened.
 
+## Preparing For a Power Cut
+
+In order to safely prepare a GEM test stand before a planned power cut execute:
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: Should be:

> In order to safely prepare a GEM test stand for a planned power cut execute:

e.g. change "before" for "for"

File: v3ElectronicsUserGuide.md [hunk 3]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
...
+
+ 1. Power down the High Voltage.
+     - Use the opportunity to power down other sensitive equipement such as PMT's.
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: Change to:

> Use the opportunity to power down other sensitive equipment, such as PMT's.

E.g. insert a comma after "equipment" and correct typo, e.g. equipment (correct) vs. equipement (wrong)

File: v3ElectronicsUserGuide.md [hunk 4]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
...
+ 2. Power down the Low Voltage.
+ 3. Place the µTCA modules in extraction mode:
+     - Gently pull the hot swap tab on the AMC's (CTP7 and AMC13). Wait until the blue LED stays on solid on each AMC's.
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: Change " on the AMC's (CTP7 and AMC13)" to "on all AMC's including the AMC13" as some users might be using GLIBs.

File: v3ElectronicsUserGuide.md [hunk 5]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
...
+     - Gently pull the hot swap tab on the MCH. Again, wait until the blue LED stays solid on.
+ 5. Power down the µTCA crate Power Modules one at a time. If powered with AC/DC converters, simply flip the switch on the AC/DC one at a time.
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: All crates will be powered with an AC/DC converter.  The converter supplies the voltage to the PM.  The difference is some manufacturers provide crates with built in converters (e.g. VadaTech) while others require an external converter (e.g. Schroff).  Try to clarify this point.

File: v3ElectronicsUserGuide.md [hunk 6]
```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
...
- 3. For each CTP7 login as `texas` and execute: `killall rpcsvc`
+ 1. Ensure the uTCA crate and associated hardware all have power.
+     - E.g. the crate, network switches, DAQ computer, etc...
+ 2. Start the DAQ computer first, then:
+     - Ensure that the `sysmgr`, `xinetd` and `dnsmasq` services are correctly started. You can use the `systemctl status <service name> command to check each service status. The `Active` field must report `active (running)`.
+     - If any of the services is not started, you can start it manually with the following command `sudo systemctl start <service name>`.
+ 3. Power on the µTCA crate. If the power cut was planned, undo the actions from the [previous section](#preparing-for-a-power-cut):
+     - Power on the µTCA crate Power Modules one at a time.
+     - Push the hot swap tab on the MCH and wait for the blue LED to turn off.
+     - Push the hot swap tabs on the AMC's and wait for the blue LED's to turn off.
```
Comment (bdorney) [inline v3ElectronicsUserGuide.md]: Change line 764 go:

> Push the hot swap tabs on the AMC's, including the AMC13, and wait for the blue LED's to turn off.

**Review by bdorney — CHANGES_REQUESTED** [review]
Slight grammatical changes needed.  Otherwise good.

**Comment by lpetre-ulb** [comment]
Grammatical changes done.

**Review by bdorney — APPROVED** [review]

**Pull request merged**